features/pytest_cov_tox.feature

```diff
@@ -0,0 +1,15 @@
+Feature: pytest-cov obeys tox
+
+    Background:
+        Given that I am in a git repository
+        And I have installed captainhook
+        And I have a tox.ini file with pytest-cov=on;0
+
+
+    Scenario: When a pytest-cov is present in tox.ini it should be obeyed.
+       When I create a file in "." called "test_one.py" containing a line over 80 chars
```

Comment:
I'm not sure this is the right test for this case. I expected to see this containing a test that created a failing test, or that created some code that wasn't tested and confirmed in either case that we did get an error.